include/LogCabin/Client.h

```diff
@@ -113,6 +113,11 @@ struct ConfigurationResult {
          * unavailable.
          */
         BAD = 2,
+        /**
+         * The reconfiguration could not be completed in the requested
+         * timeframe.
```

Comment:
might be my unfamiliarity with doxygen but it's unclear to me where you can and cannot put the \since tag. in the previous commit I added \since to a new parameter, but it appeared as if I was saying the whole function was since 1.2. regardless, have added this tag.